### DISCUSSION TITLE
SageTV Forums

### DISCUSSION THREAD

**Discussion by Gary0201** [OpenSageTV/Community#58] [Linux]
Tried to view some of the SageTV Forum posts on installing on Linux, and links are failing.  Looks like much of the old forum simply isn't viewable anymore.

**Comment by Sagetvmaster** [comment]
yes, I just checked in there - I was under the impression that Narflex said we'd be able to use it as a 'searchable' entity, i.e. you could still look for answers to things, but couldn't sign-in or leave messages, enter threads etc.
But now, it's just returning a 404 error.
Anyone of the old admins/higher-ups know what's going on with this?
Thanks

**Comment by Ugadata** [comment]
The old site doesn't work with PHP any more (the search results come back with php referencing) 

It has been indicated that the results can be massaged to return the correct results.

There is another post (or maybe more) that suggest what can be done.

[Here's one](https://github.com/OpenSageTV/Community/discussions/1)

**Reply by Sagetvmaster** [reply]
Thanks - I checked that post but found no solution.  What exactly do you mean by 'messaged'?
All I was looking to do was search within it, which did work after June 1st, but does not at the moment..

**Reply by eelpout** [reply]
that was my post and it's buried in the thread. 

you have to change to point to the actual HTML page. for example, the following Google result:

https://forums.sagetv.com/forums/showthread.php?t=62774

you'll need this to change to this:

https://forums.sagetv.com/forums/showthread_t_62774.html

---

we really need to get the static site reindexed for the html pages. but who owns it now?

**Reply by Sagetvmaster** [reply]
I actually downloaded the entire site on Narflex’s instructions, but even it is not searchable now.  Somehow, the download must be pointing back to the HTML site for some reason.  Not sure who officially owns it though…

**Reply by Ugadata** [reply]
it was supposed to be massaged not messaged

**Reply by LoisGNS** [reply]
I can search my local copy of the static (from httrack), but the results look something like this (in the Content view mode in Windows File Explorer)

![image](https://github.com/OpenSageTV/Community/assets/76415227/b33de3e8-9006-4a09-96ab-1a39b3c70c8d)


So I have to open each and every one to see if it's what I was looking for, but the internal links on those posts work ok.
If I use Google or otherwise  visit the static site online, the results are more helpful in picking what's most relevant to my search, e.g.:

![image](https://github.com/OpenSageTV/Community/assets/76415227/a163d284-6603-4d92-b44a-a808638eca17)

But then I have to massage the output to make it viewable. Also painful to do one at a time, but the internal links do work ok.

So it would be nice to have some utility to approach it from one of the two directions: 
(1) autoconvert the search-hits to the format needed. The transformation is to change one string to another, but you have to do it to however many search hits you have on hand - or the ones you want to follow up on
or
(2) something that will take the hits found on the local HTTRACK'd static copy and expand them to show enough content to decide which ones are of interest. From the screen-grab, some of the info is present, but a few more lines would be a big help. 

I tried the preview pane, but that showed raw HTML or refused to display at all because the size of most of the html files was larger than it wanted to show. 

I'm running some searches on potential utilities for this, but if anyone already uses something they recommend, please chime in.
